.gitignore

```diff
@@ -13,3 +13,7 @@
 /.php-cs-fixer
 /.php-cs-fixer.cache
 ###< friendsofphp/php-cs-fixer ###
+
+###> PHPStorm IDE ###
+.idea
+###< PHPStorm IDE ###
```

Comment:
please do those platofrm specific ignores in your global gitignore configuration on your system ;-)